test/fixtures/samples/jest/__tests__/jest.js

```diff
@@ -44,7 +44,6 @@ it('valid-expect good', () => {
 });
 
 // `global-require` - Allow scoped require
-// `prefer-import/prefer-import-over-require` - Ignore imports
```

Comment:
If you agree with what I said about about Jest, please restore this line and similar ones but with the new thing.